app/helpers/claims_helper.rb

```diff
@@ -58,4 +58,12 @@ def payment_answers(claim)
   def date_of_birth_string(claim)
     claim.date_of_birth && l(claim.date_of_birth)
   end
+
+  def format_claim_reference(claim)
```

Comment:
There is actually two `claim_helper.rb` files: one in the root of the `helpers` folder and another namespaced under `admin`. Because this method is used in the admin area of the application I reckon it belongs in the `admin/claim_helper.rb` file.

Also wondering if we can give this method a more appropriate (intention-revealing) name. "format claim reference" sounds like a method that would format a reference. It feels like this method's responsibility is actually to give us a _heading for the claim summary_, which happens to be a claim reference for undecided claims, and a claim reference with an outcome for decided claims. Maybe something like `claim_summary_heading` would better describe what this method gives us?

Reply:
I agree, `claim_summary_heading` is more descriptive. I have made this change.